client/src/components/pages/application/Project/index.js

```diff
@@ -1,5 +1,139 @@
+  const [userName, setName] = useState('');
+  const history = useHistory();
+
+  const throwMessage = (msg) => setMessage(msg);
+
+  useEffect(() => {
+    getUserID().then((data) => {
+      if (data.message === 'you are authorized') {
+        setId(data.userId);
+      }
+    });
+    if (UserId) {
+      axios.get(`/api/v1/applicants/${UserId}`).then(({ data: { user } }) => {
+        setGithubLink(user.githubLink);
+        setName(user.fullName);
+        projectData(user.projectId).then((project) => {
+          setProjectTitle(project.projectName);
+          setProjectDesc(project.projectInstructions);
+        });
+      });
+    }
+  }, [UserId]);
+
+  // useEffect(() => {
+  //   axios.get(`/api/v1/project/${projectId}`).then(({ data: { project } }) => {
+  //     setProjectTitle(project.projectName);
+  //     setProjectDesc(project.projectInstructions);
+  //   });
+  // });
+
+  const Next = () => {
+    FProjectValidation({ githubLink })
+      .then(() =>
+        axios.patch(`/api/v1/applicants/${UserId}`, {
+          githubLink,
+        })
+      )
+      .then(() => history.push('/submit'))
+      .catch(({ errors }) => throwMessage(errors));
+  };
+
+  return (
+    <div className="Container_page">
+      <Helmet>
+        <title>Final Project</title>
+      </Helmet>
+      <img src={backGround} alt="backGround" className="backGround" />
+      <div className="text_Welcome">
+        <Typography variant="h3" color="default">
+          Welcome, {userName}
+        </Typography>
+      </div>
+      {projectTitle && projectDesc ? (
+        <div className="Form_container">
+          <Typography variant="h5">Final Project</Typography>
+          <>
+            <div className="label_container">
+              <Typography className="label">Project Title</Typography>
+            </div>
+            <TextField name="projectTitle" value={projectTitle} />
+          </>
+          <>
+            <div className="label_container">
+              <Typography className="label">Project Description</Typography>
+            </div>
+            <TextareaAutosize
+              aria-label="minimum height"
+              rowsMin={11}
+              className="text_area"
+              value={projectDesc}
+            />
+          </>
+          <>
+            <div className="label_container">
+              <Typography className="label">Project Link in Github</Typography>
+            </div>
+            <TextField
+              name="githubLink"
+              value={githubLink}
+              onChange={(e) => setGithubLink(e.target.value)}
+              placeholder="Enter Your Github Link"
+              isError={
+                message.includes('Enter GitHub link') ||
+                message.includes('Github Not Match')
+              }
+              message={
+                message.includes('Enter GitHub link') ||
+                (message.includes('Github Not Match') && 'Error Github link')
```

Comment:
this one is enough , I think 
```suggestion
                (message.includes('Error in Github link')
```